src/disclosure-manager/index.js

```diff
@@ -1,2 +1,14 @@
-export { default } from 'terra-disclosure-manager';
-export * from 'terra-disclosure-manager';
+import DisclosureManager, {
+  disclosureManagerShape, availableDisclosureSizes, availableDisclosureHeights, availableDisclosureWidths,
+} from './DisclosureManager';
+import withDisclosureManager from './withDisclosureManager';
+import DisclosureManagerContext from './DisclosureManagerContext';
+import DisclosureManagerHeaderAdapter from './DisclosureManagerHeaderAdapter';
+import DisclosureManagerHeaderAdapterContext from './DisclosureManagerHeaderAdapterContext';
+import DisclosureContainer from './DisclosureContainer';
```

Comment:
Should we provide this container to consumers or is this an internal implementation detail? Is the expected usage when a custom registerDismissCheck is used?

Reply:
We could hide it from the index file, I don't expect people to use this directly.

Reply:
Do we also want to prefix the filename with an underscore to indicate that it's private?

Reply:
I can make that happen